### ISSUE TITLE
Apollo 1.8 - "Share ... as an image" places the image too low on the screen. iPados 13.5, iPadPro

### ISSUE THREAD

**Issue by altegthb** [christianselig/apollo-bugs#742]
![20670045-97E1-4CB2-8F98-62379EE5A385](https://user-images.githubusercontent.com/65847877/82749387-a0287680-9db1-11ea-8c89-e25ad6bc6545.png)
**Describe the bug**

**To Reproduce**

**iOS version**: Version of iOS this bug occurred on (can be found in Settings.app > General)
**Apollo version**: Version of Apollo this bug occurred on (can be found in Apollo's Settings tab > About)
**iOS device**: For example iPhone XS, iPad mini, etc.
**Jailbroken**: Whether or not your device is jailbroken
**Reproducible:** Can you reproduce it every time? Or is it random?

**Expected behavior**

**Screenshots**
If possible, add screenshots to help show the problem.

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**

**Comment by altegthb** [comment]
The SHARE button is out of screen. Updated to version 1.8.1. THE SAME BUG!

**Comment by nguyenhm16** [comment]
Yeah just hit this bug. Still there in 1.8.4, iPadOS 13.5.1, 2018 iPad Pro 11”. Kinda annoying. 
![BF0A5979-DD22-4D8E-8952-76185A988AAE](https://user-images.githubusercontent.com/38112286/86864614-1ef01f00-c093-11ea-8fd2-399648884821.png)
